sql/moz-fx-data-shared-prod/fenix_derived/google_ads_campaign_cost_breakdowns_v1/query.sql

```diff
@@ -0,0 +1,76 @@
+WITH campaigns AS (
+  SELECT DISTINCT
+    id,
+    name,
+    FORMAT("%s (%s)", name, CAST(id AS string)) AS felix_compatible_campaign_name
+  FROM
+    `moz-fx-data-bq-fivetran`.google_ads.campaign_history
+),
+install_dou_metrics AS (
+  SELECT
+    fenix_marketing_metrics.adjust_campaign AS fenix_marketing_metrics_adjust_campaign,
+    fenix_marketing_metrics.cohort_date AS date,
+    COALESCE(SUM(fenix_marketing_metrics.new_profiles), 0) AS new_profiles_sum,
+    COALESCE(SUM(fenix_marketing_metrics.dau), 0) AS dau_sum
+  FROM
+    `moz-fx-data-shared-prod.fenix.marketing_attributable_metrics` AS fenix_marketing_metrics
+  GROUP BY
+    1,
+    2
+)
+SELECT
+  campaigns.name AS campaign_name,
+  stats.date AS date,
+  -- Total spend per-campaign
+  stats.cost_micros / 1000000.0 AS campaign_spend_in_cents,
```

Comment:
wdyt about keeping this micros like the rest of them, and then changing it to $ in the view? (int > float every time)

Reply:
Happy to do that!